### DISCUSSION TITLE
Hello everyone

### DISCUSSION THREAD

**Discussion by aabdurakhmanov** [AAbdurakhmanov/TTS_NLP_python_graduate_work#4] [Ideas]
Hello everyone, please give me some ideas to improve my little project, thanks in advance

**Comment by aabdurakhmanov** [comment]
Something new

**Reply by FrankBenjamen** [reply]
Hi bro i think you need add to your project some AI algoritms speech recognition and Voice recognition

**Comment by FrankBenjamen** [comment]
Hi bro i think you need add to your project some AI algoritms speech recognition and Voice recognition
